src/CommunityToolkit.Aspire.Client.MassTransit.RabbitMQ/MassTransitClientExtensions.cs

```diff
@@ -0,0 +1,78 @@
+﻿using MassTransit;
+using MassTransit.Monitoring;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using System.Reflection;
+
+namespace CommunityToolkit.Aspire.Client.MassTransit.RabbitMQ;
+
+/// <summary>
+/// MassTransitClientExtensions provides extension methods for configuring MassTransit in a client application.
+/// </summary>
+public static class MassTransitClientExtensions
+{
+    /// <summary>
+    /// Configures MassTransit with RabbitMQ integration for the client side,
+    /// using the same configuration used by the hosting environment.
+    /// </summary>
+    /// <param name="services">The client service collection.</param>
+    /// <param name="name">A unique name for the RabbitMQ instance.</param>
+    /// <param name="telemetry">Enables telemetry, which could be exported to either OpenTelemetry or Application Insights.</param>
+    public static void AddMassTransitClient(this IServiceCollection services, string name, bool? telemetry = false)
+    {
+        if (string.IsNullOrWhiteSpace(name))
+            throw new ArgumentException("The name must be a valid, non-empty string.", nameof(name));
+
+        var configuration = services.BuildServiceProvider().GetRequiredService<IConfiguration>();
+        var configurationSection = configuration.GetSection($"MassTransit:{name}");
+        var options = new MassTransitOptions();
```

Comment:
For options management, we need to provide an `Action` callback that can be optionally provided to perform configuration in code rather than via config: https://github.com/CommunityToolkit/Aspire/blob/main/src/CommunityToolkit.Aspire.OllamaSharp/AspireOllamaSharpExtensions.cs#L40

Check out how https://github.com/CommunityToolkit/Aspire/blob/main/src/CommunityToolkit.Aspire.OllamaSharp/AspireOllamaSharpExtensions.cs is implemented for more insights.